samples/samples-powershell/host.json

```diff
@@ -0,0 +1,15 @@
+{
+  "version": "2.0",
+  "logging": {
+    "applicationInsights": {
+      "samplingSettings": {
+        "isEnabled": true,
+        "excludedTypes": "Request"
+      }
+    }
+  },
+  "extensionBundle": {
+    "id": "Microsoft.Azure.Functions.ExtensionBundle.Preview",
```

Comment:
Did you verify that the tests are using the build extension DLL and not the one from the bundle?

Reply:
Verified and see Lucy's comment.